tests/mountingOptions/stubs.global.spec.ts

```diff
@@ -37,6 +37,29 @@ describe('mounting options: stubs', () => {
     expect(wrapper.html()).toBe('<div></div><foo-stub></foo-stub>')
   })
 
+  // https://github.com/vuejs/vue-test-utils-next/issues/249
+  it('applies stubs globally', () => {
+    const Comp = defineComponent({
+      template: '<div><router-link /><router-view /></div>'
+    })
+    const wrapper = mount(Comp, {
+      global: {
+        stubs: {
+          RouterLink: true,
+          RouterView: defineComponent({
+            render() {
+              return h('span')
+            }
+          })
+        }
+      }
+    })
+
+    expect(wrapper.html()).toBe(
+      '<div><router-link-stub></router-link-stub><span></span></div>'
+    )
```

Comment:
As I'm a bit unclear on how everything is related, can you maybe add a test where we try to find the component registered as a stub? I'd like to be sure that `findComponent` and friends work in that case as well.

I think a real use-case would be to stub `RouterLink` with `RouterLinkStub`, find it, and check its props to see if the component is creating links with the appropriate values. So I'd like to make sure we cover this 😅

Reply:
Good idea

Reply:
Done: https://github.com/vuejs/vue-test-utils-next/pull/260/commits/20c76940652cb64a4dab4b6bfd4ce45e4ca30ea9

Is this what you mean?

Reply:
Perfect, thanks Lachlan!

Reply:
Cool, want to give me a big old ✅